cmd/test-external-check/README.md

```diff
@@ -1,16 +1,18 @@
 ### test-external-check
 
-This is a container for use in testing external checks on Kuberhealthy.  This container simply runs and either reports a pass or fail depending on the `REPORT_FAILURE` environment variable in its spec.
+This is a container used to test external checks in Kuberhealthy. This container simply runs, waits for a set interval of seconds and then either reports a success or failure depending on the `REPORT_FAILURE` environment variable set in the pod spec.
 
-##### 2.0.0 alpha setup
+##### test-external-check Setup
 
-The version 2.0.0 alpha for Kubernetes image is: `quay.io/comcast/kuberhealthy:2.0.0alpha`.  
+To setup external checks on Kuberhealthy 2.0.0 the container for Kubernetes can be found at: `quay.io/comcast/kuberhealthy:2.0.0`.  
 
-You will need to run this image of kuberhealthy in your cluster before external checks are available to you.
+The external checks are only in available in Kuberhealthy 2.0.0 and later.
 
-You will also need to define the following CRD for Kuberhealthy 2 to use.
+To begin using the external checks be sure that the `khchecks.comcast.github.io` CRD is already applied to your cluster.
```

Comment:
nitpicky sorry! just add a comma --> `To begin using the external checks, be sure`

Reply:
done as well